lib/pages/chat/events/message/message.dart

```diff
@@ -217,6 +217,7 @@ class _MessageState extends State<Message> {
           children: rowChildren,
         );
         return Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
```

Comment:
Can you demo the gesture: swipe to reply?

Reply:
[2074swipe.webm](https://github.com/user-attachments/assets/6b16cafc-8571-47b8-b631-38adbd37bc9d)